File: pascals-triangle-ii/pascals-triangle-ii.py
class Solution:
    def getRow(self, rowIndex: int) -> List[int]:
        '''
        Generates the row in Pascal's triangle to return
        Returns the requested row of Pascal's Triangle
        '''
        answer = [1]
        for i in range(1, rowIndex+1):
            spot = self.factorial(rowIndex) // (self.factorial(i) * self.factorial(rowIndex - i))
            answer.append(spot)
        return answer
    
    def factorial(self, n: int) -> int:
        val = 1
        for i in range(1, n+1):
            val *= i
        return val

File: pascals-triangle-ii/pascals-triangle-ii.java
class Solution 
{
    public List<Integer> getRow(int rowIndex) 
    {
        // Create row to store answer
        List<Integer> answer = new ArrayList<>();
        answer.add(1);
        
        // Perform combinations to generate each Pascal column in the given row
        for (int col = 1; col <= rowIndex; col++)
            answer.add((int)((long)answer.get(answer.size()-1) * (rowIndex-(col-1))/col));
        
        return answer;
    }
}